docs/wagtail-pages.md

```diff
@@ -0,0 +1,211 @@
+# Wagtail pages
+
+[Wagtail pages](http://docs.wagtail.io/en/v1.13.4/topics/pages.html) are 
+[Django models](https://docs.djangoproject.com/en/1.11/topics/db/models/) 
+that are constructed of 
+[fields](#fields), [StreamFields](#streamfields), and [panels](#panels) 
+that are rendered in a standard way. 
+All CFPB Wagtail pages should inherit from the 
+[`v1.models.base.CFGOVPage` class](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/base.py).
+
+!!! note
+    Before creating a new Wagtail page type 
+    please consider whether one of our existing page types can meet your needs, 
+    whether this is a different type of content than we have anywhere else in the site, 
+    and whether there is maintenance efficiency gained from a new page type.
+
+There are types of information defined on a new Wagtail page model: 
+basic [database fields](#fields) (like any Django model), 
+specialized database fields called [StreamFields](#streamfields) that allow for freeform page content, 
+and [editor panels](#panels) that present these fields to content editors. 
+
+## Fields
+
+Database fields in Wagtail pages work exactly the same as in 
+[Django models](https://docs.djangoproject.com/en/1.11/topics/db/models/#fields), 
+and Wagtail pages can use any [Django model field](https://docs.djangoproject.com/en/1.11/ref/models/fields/). 
+
+For example, our `BrowsePage` 
+[includes a standard Django `BooleanField`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/browse_page.py) 
+that allows content editors to toggle secondary navigation sibling pages:
+
+```python
+from django.db import models
+
+from v1.models.base import CFGOVPage
```

Comment:
It seems a bit unnecessary to include the imports here when you're already heavily subsetting the class definition, but I defer to your experience and expertise.

Reply:
I like to have the relevant imports. `models` is shown and used, and `CFGOVPage` is too.